apps/web/env.mjs

```diff
@@ -70,6 +71,7 @@ export const env = createEnv({
     NEXT_PUBLIC_POSTHOG_API_KEY: z.string().optional(),
     NEXT_PUBLIC_POSTHOG_API_HOST: z.string().optional(),
     NEXT_PUBLIC_SENTRY_DSN: z.string().optional(),
+    NEXT_PUBLIC_SURVEY_BASE_URL: z.string().optional(),
```

Comment:
Please use the existing serverside env variables (`SURVEY_BASE_URL`) and pass them through to the client component. We can’t use NEXT_PUBLIC_ env variables because we need all env variables to be evaluated at runtime to support the docker deployment.